solo/cli/update.py

```diff
@@ -43,190 +43,9 @@
     help="Development option: use release refered to by ALPHA_VERSION",
 )
 def update(serial, yes, local_firmware_server, alpha):
-    """Update Solo key to latest firmware version."""
+    """Update Nitrokey key to latest firmware version."""
 
     update_url = 'https://update.nitrokey.com/'
     print('Please use {} to run the firmware update'.format(update_url))
     return
 
-    # Determine target key
-    try:
-        solo_client = solo.client.find(serial)
-
```

Comment:
I see the update procedure source code is removed here. It should be reinstated in the next releases though.